en/docs/learn/configuring-oauth2-openid-connect.md

```diff
@@ -52,7 +52,9 @@ the users with an authorization server-based authentication.
 
         ```toml
         [oauth]
-		hash_tokens_and_secrets = true 
+		hash_tokens_and_secrets = true
```

Comment:
Does this render properly?

Reply:
<img width="663" alt="Screenshot 2022-05-13 at 17 45 39" src="https://user-images.githubusercontent.com/40081059/168281256-694677a3-d95c-4960-bd10-e2f97a6b1f93.png">

**Renders properly**